src/main/java/io/nats/client/JetStreamSubscription.java

```diff
@@ -56,9 +56,12 @@
     void pullExpiresIn(int batchSize, Duration expiresIn);
 
     /**
+     * Fetch (pull) the list
      *
      * @param batchSize the size of the batch
      * @param timeout the timeout
+     *
+     * @return the list of messages
      */
     List<Message> fetch(int batchSize, Duration timeout);
```

Comment:
Did you intend to include fetch in this PR?

Reply:
I fixed the javadoc, this was already in master